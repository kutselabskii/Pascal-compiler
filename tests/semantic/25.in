program abc;

var a, i: integer;

begin
for i := 1.5 downto 5 do
	a := a + i;
end.