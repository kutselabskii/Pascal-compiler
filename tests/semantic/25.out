Line 6 : Col 25 : Semantir error :: 'for' starting and finishing expressions should have 'integer' values
